### PR TITLE
Feature/ngui pausedeploy

### PR DESCRIPTION
**Pull request by gaarf**
https://issues.cask.co/browse/COOPR-555 + other minor tweaks

### CODE REVIEW

**Comment by gaarf** [comment]
screen shot 2014-10-29 at 1 02 53 pm

File: coopr-ngui/app/js/controllers/cluster/detail-ctrl.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ function ($scope, MYSERVICEPICKER_EVENT, CrudFormBase, $state, $modal, $alert, m
     $scope.additionalServices = [];
 
     if(data.status === 'pending') {
-      timeoutPromise = $timeout(update, 1000);
+      timeoutPromise = $timeout(update, 10000);
```
Comment (ajainarayanan) [inline coopr-ngui/app/js/controllers/cluster/detail-ctrl.js:46]: Any specific reason why the increase in the timeout? Seems to be changed uniformly in the cluster feature.

Comment (gaarf) [inline coopr-ngui/app/js/controllers/cluster/detail-ctrl.js:46]: yes, talked to @wolf31o2 , in real conditions no task would take less than 30seconds. So updating every 10seconds seems like enough.

**Comment by ajainarayanan** [comment]
:+1: LGTM.

**Pull request merged**